### PR TITLE
Update footer links for better dev onboarding

### PR DESCRIPTION
**Pull request by dcts**
This change enhances the onboarding process for new developers by providing direct links to "help wanted" issues, reducing the overwhelm of browsing through unlabeled issues.

### CODE REVIEW

**Comment by yuliaSharabi** [comment]
@dcts Can you link the preview? I can review this...

**Comment by dcts** [comment]
@yuliaSharabi the deploy preview just got posted. 

But in any case I need a review from a maintainer to merge, so I assigned @Radstakes :)

**Review by Radstakes — APPROVED** [review]

**Pull request merged**
